config/config.go

```diff
@@ -552,7 +552,7 @@ func (c *Config) LoadConfigData(data []byte) error {
 	if c.Agent.SnmpTranslator == "netsnmp" {
 		PrintOptionValueDeprecationNotice("agent", "snmp_translator", "netsnmp", telegraf.DeprecationInfo{
 			Since:     "1.25.0",
-			RemovalIn: "2.0.0",
+			RemovalIn: "1.40.0",
```

Comment:
Should this be 1.35.0?

Reply:
no - I meant to talk about this one in pairs, but no way we should change this next year.